src/Apps/Collect2/Routes/Collection/index.tsx

```diff
@@ -121,13 +121,17 @@ export class CollectionApp extends Component<CollectionAppProps> {
               />
             </ArtworkFilterContextProvider>
           </Box>
-          <Separator mt={6} mb={3} />
-          <Box mt="3">
-            <RelatedCollectionsRail
-              collections={viewer.relatedCollections}
-              title={viewer.title}
-            />
-          </Box>
+          {viewer.linkedCollections.length === 0 && (
```

Comment:
```suggestion
          {!showCollectionHubs && (
```

We define a variable [on line 63](https://github.com/artsy/reaction/pull/2942/files#diff-65af23e87805db8cca6b1a5d0000c390R63) that gives a good hint about what it means for `viewer.linkedCollections.length` to equal 0. I think it'd make sense to use that, instead of doing another check against `linkedCollections.length`.

Reply:
Yeah I considered that but since `showCollectionHubs` also includes the admin check because your [PR](https://github.com/artsy/reaction/pull/2940/files) isn't merged, I figured this was okay.